### PR TITLE
feat(packagers): add basic yarn2 without pnp 

### LINKED ISSUE #1244: Add e2e test for yarn2 without pnp support
https://github.com/prisma/prisma/issues/4493

### PR DESCRIPTION
**Pull request by williamluke4**
closes https://github.com/prisma/e2e-tests/issues/1244

### CODE REVIEW

**Review by timsuchanek — APPROVED** [review]

File: packagers/yarn2-without-pnp/script.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { PrismaClient } from '@prisma/client'
+
+const client = new PrismaClient()
+
+async function main() {
+  await client.user.deleteMany({})
+
+  const rand = Math.random().toString()
```
Comment (divyenduz) [inline packagers/yarn2-without-pnp/script.ts:8]: Why is this needed when you are using `deleteMany`? We don't do this in any of the other tests yet if your aim is idempotent tests https://github.com/prisma/e2e-tests/issues/991

**Review by divyenduz — APPROVED** [review]
Approved with a comment

File: packagers/yarn2-without-pnp/script.ts (hunk 1 above)
Comment (williamluke4) [reply packagers/yarn2-without-pnp/script.ts:8]: Hey @divyenduz, This test is just a modified version of the yarn test, which uses the exact same `script.ts`. I will change it if you think it's necessary
